api/docs/release.dox

```diff
@@ -134,10 +134,22 @@ Further non-compatibility-affecting changes include:
 **************************************************
 <hr>
 
+The changes between version 11.3.0 and 11.2.0 include the following compatibility
+changes:
```

Comment:
These do not seem like compatibility issues at all: shouldn't these be non-compatibility-affecting changes and there aren't any compat changes?

Reply:
Agreed. Moved to non-compatibility-affecting changes.